poprox-db/migrations/versions/2025_06_07_0854-87aa4d7933b2_create_an_experiences_table.py

```diff
@@ -46,7 +47,7 @@ def upgrade() -> None:
 
 
 def downgrade() -> None:
-    op.drop_constraint("fk_top_stories_team_id", "top_stories", type_="foreignkey")
-    op.drop_constraint("fk_experiences_recommender_id", "top_stories", type_="foreignkey")
+    op.drop_constraint("fk_experiences_team_id", "experiences", type_="foreignkey")
```

Comment:
I think this also needs to change on line 41 above

Reply:
You're probably right. I'm surprised the tests din't catch that one. I wonder if our database schema tests are no longer automatic.